mlflow/java/spark/src/main/scala/org/mlflow/spark/autologging/DatasourceAttributeExtractor.scala

```diff
@@ -49,10 +49,20 @@ private[autologging] trait DatasourceAttributeExtractorBase {
       deltaInfoOpt
     } else {
       leafNode match {
+        // DataSourceV2Relation was removed in Spark 3.0.0 stable release due to some issue and 
+        // still not present in Spark 3.2.0. While we are not sure whether it will be back in
+        // the future, we still keep this code here to support previous versions.
         case relation: DataSourceV2Relation =>
           getSparkTableInfoFromTable(relation.table)
-        case other =>
-          None
+        // This is the case for Spark 3.x except for 3.0.0-preview
+        case LogicalRelation(HadoopFsRelation(index, _, _, _, fileFormat, _), _, _, _) =>
+          val path: String = index.rootPaths.headOption.map(_.toString).getOrElse("unknown")
```

Comment:
Spark can read from multiple paths in one shot. Do we want to display all of them or only the first path?